app/models/customer.py

```diff
@@ -1,4 +1,31 @@
 from app import db
+from flask import current_app
+from sqlalchemy.orm import backref
+from app.models.rental import Rental
+
 
 class Customer(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime)
+    
+    videos_checked_out_count = db.Column(db.Integer)
+    # backref is creating 2 new attributes as well as establishing the relationship here
+    # backref is creating Customer.videos as well as Video.customers    
+    # updated video and customer with cascade
+    # When the customer is deleted, all associated rentals are also deleted
+    # cascade allows us to "delete the ophans" associated with the customer id 
+    # the videos that the customer has are accessed through the rental table
+    # the rental is the child of the customer
+    # orphan cascade is normally configured only on the "one" side of a one-to-many relationship, and not on the "many" side of a many-to-one or many-to-many relationship.  To force this relationship to allow a particular "Video" object to be referred towards by only a single "Customer" object at a time via the Customer.videos relationship, which would allow delete-orphan cascade to take place in this direction, set the single_parent=True flag. (Background on this error at: http://sqlalche.me/e/13/bbf0)"
```

Comment:
Even though a bunch of comments is not necessarily a bad thing it could be moved to a ReadMe file to reference big blocks of comments for readability.